addon/index.js

```diff
@@ -346,9 +346,13 @@ function updateExperiments() {
 function syncAllAddonInstallations(serverInstalled) {
   const availableIDs = Object.keys(store.availableExperiments);
   const clientIDs = Object.keys(store.installedAddons);
-  const serverIDs = serverInstalled
-    .filter(item => item.client_id === store.clientUUID)
-    .map(item => item.addon_id);
+  const serverIDs = [];
```

Comment:
Something we could do short of versioning the API for now: That `serverInstalled` data [comes from the web frontend](https://github.com/mozilla/testpilot/blob/master/testpilot/frontend/static-src/app/models/me.js#L45) in the `sync-installed` message. On the frontend, we could massage that installation object into the array the existing add-on version expects, and hold off on the add-on change until/unless we get a versioned API